libs/platform/src/lib/table-helpers/directives/table-virtual-scroll.directive.ts

```diff
@@ -68,6 +128,14 @@ export class TableVirtualScrollDirective extends TableVirtualScroll implements O
         if (this.virtualScroll && (changes['rowHeight'] || changes['virtualScroll'] || changes['renderAhead'])) {
             this.calculateVirtualScrollRows();
         }
+        if (changes?.scrollWholeRows && changes.scrollWholeRows.currentValue && !this.virtualScroll) {
```

Comment:
i think if we are going to exclude ngOnChanges, it will be better if you use input set or input signal instead of handling this logic inside ngOnchanges.

Reply:
No signals in ng15 but yes an input setter will work